bin/start-dashboard.sh

```diff
@@ -26,16 +26,14 @@ load_rss_env
 
 cd "$RSS_HOME"
 
-COORDINATOR_CONF_FILE="${RSS_CONF_DIR}/coordinator.conf"
+DASHBOARD_CONF_FILE="${RSS_CONF_DIR}/dashboard.conf"
```

Comment:
At present, there is no `dashboard.conf` in the tarball. Should we add a default file for this?